### PR TITLE
Adding the Json Parser Object as the Class Member Variable

### PR DESCRIPTION
**Pull request by aditjind**
*Fixes #:* Instead of creating a new JsonParser() variable every-time creating one which will load whenever the Action Handler loads. 

*Description of changes:* Adding the Json Parser Object as the Class Member Variable

*Tests:*

*If new tests are added, how long do the new ones take to complete*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rest/QueryActionRequestHandler.java [hunk 1]
```diff
@@ -73,11 +72,13 @@
 public class QueryActionRequestHandler extends MetricsHandler implements HttpHandler {
     public static final String ACTION_SET_JSON_NAME = "LastSuggestedActionSet";
     private static final Logger LOG = LogManager.getLogger(QueryActionRequestHandler.class);
+    public JsonParser jsonParser;
```
Comment (sidheart) [inline src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rest/QueryActionRequestHandler.java:75]: This doesn't need to be public

**Review by sidheart — APPROVED** [review]

**Review by sidheart — DISMISSED** [review]

**Review by sruti1312 — APPROVED** [review]

**Comment by yojs** [comment]
@aditjind There are 6 test failures. It might be good to rebase and re-trigger the CI ?

**Review by aditjind — COMMENTED** [review]
> @aditjind There are 6 test failures. It might be good to rebase and re-trigger the CI ?

Done. Failures are a subset of the flaky tests

**Review by ricardolstephen — APPROVED** [review]

**Comment by yojs** [comment]
can we add in the description of the PR; the motivation for the change ?

**Comment by aditjind** [comment]
> can we add in the description of the PR; the motivation for the change ?

Done

**Review by sidheart — APPROVED** [review]

**Review by sruti1312 — APPROVED** [review]

**Pull request merged**